foreign_cc/private/framework.bzl

```diff
@@ -767,13 +767,23 @@ def _symlink_contents_to_dir(dir_name, files_list):
         path = _file_path(file).strip()
         if path:
             lines.append("##symlink_contents_to_dir## \
-$$EXT_BUILD_ROOT$$/{} $$EXT_BUILD_DEPS$$/{} True".format(path, dir_name))
+$$EXT_BUILD_ROOT$$/{} $$EXT_BUILD_DEPS$$/{} True".format(path, _get_dir_name(dir_name, file)))
 
     return lines
 
 def _file_path(file):
     return file if type(file) == "string" else file.path
 
+def _get_dir_name(dir, file):
+    if dir == "include":
```

Comment:
Don't we always want to do this? Surely if it was linking say share/ into the tree you'd also want the folder structure to be maintained?